bash_completionsV2.go

```diff
@@ -173,8 +169,9 @@ __%[1]s_process_completion_results() {
         __%[1]s_handle_completion_types
     fi
 
-    __%[1]s_handle_special_char "$cur" :
-    __%[1]s_handle_special_char "$cur" =
+    __%[1]s_handle_wordbreaks "$cur"
```

Comment:
At surface value, this line looked wrong to me because I didn't think it was correct to handle wordbreaks like this when the user requested menu-completion. But after some manual testing I'm pleasantly surprised to find that this is correct, and menu-completion does still expect the same behavior as "regular" completion.